### PR TITLE
fix typo, the the, becasue, the life the task

### PR DESCRIPTION
**Pull request by sunghan-chang**
1. the the -> the
2. the life the task group -> the life of the task group
3. becasue -> because

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

File: os/drivers/usbhost/usbhost_trace.c [hunk 1]
```diff
@@ -127,7 +127,7 @@ static int usbhost_trsyslog(uint32_t event, FAR void *arg)
 		syslog(LOG_INFO, fmt, (unsigned int)TRACE_DECODE_U23(event));
 	}
 
-	/* No, then it must the the two argument format first. */
+	/* No, then it must the two argument format first. */
```
Comment (jeongchanKim) [inline os/drivers/usbhost/usbhost_trace.c:130]: There is no verb. I think this should be 
`then it must be the two ...`

**Review by davidfather — APPROVED** [review]
lgtm.

**Review by sangwon03 — APPROVED** [review]
LGTM

**Review by sangam-swami — APPROVED** [review]
looks good

**Pull request merged**